environments/helm-values/values-int.yaml

```diff
@@ -0,0 +1,98 @@
+###############################################################
+# Copyright (c) 2024 BMW Group AG
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+###############################################################
+
+ingress:
+  enabled: true
+  className: "nginx"
+  annotations:
+    cert-manager.io/cluster-issuer: "letsencrypt-prod"
+    nginx.ingress.kubernetes.io/use-regex: "true"
+    nginx.ingress.kubernetes.io/enable-cors: "true"
+    nginx.ingress.kubernetes.io/proxy-body-size: "8m"
+    nginx.ingress.kubernetes.io/cors-allow-origin: "https://*.int.catena-x.net"
+  tls:
+    - secretName: "dim.int.catena-x.net-tls"
+      hosts:
+        - "dim.int.catena-x.net"
+  hosts:
+    - host: "dim.int.catena-x.net"
+      paths:
+        - path: "/api/dim"
+          pathType: "Prefix"
+          backend:
+            port: 8080
+
+dim:
+  swaggerEnabled: true
+  rootDirectoryId: "27fee02a-e265-4cfc-af70-4f217a33840b"
```

Comment:
this should be changed to the sap provided value for the newly created directory. maybe change it to tbd for now?

Reply:
thanks for clarifying! same for operatorId below, right?